docs/LSPSupport.md

```diff
@@ -458,6 +458,67 @@ Here is an example with the [Java Language Server](https://github.com/eclipse-jd
 
 ![textDocument/onTypeFormatting](./images/lsp-support/textDocument_onTypeFormatting.gif)
 
+If desired &mdash; for example, for those who want to control when formatting is performed &mdash; server-side/LSP-based
+on-type formatting can be disabled via client configuration as follows:
+
+```json
+{
+  "format": {
+    "onTypeFormatting": {
+      "serverSide": {
+        "enabled": false
+      }
+    }
+  }
+}
+```
+
+#### Client-side On-Type Formatting

```

Comment:
And I documented client-side on-type formatting extensively including the TypeScript server configuration example and a demo.